truststore_linux.go

```diff
@@ -17,7 +17,8 @@ var (
 )
 
 func (m *mkcert) installPlatform() {
-	log.Fatalf("-install is not yet supported on Linux 😣\nYou can manually install the root certificate at %q in the meantime.", filepath.Join(m.CAROOT, rootName))
+	log.Println("  -install is not yet fully supported on Linux")
```

Comment:
What do you have against emojis 😁

Reply:
Whoops. Fixed. They don't show up the best in my emacs config. :-/